02week/pigLatin.js

```diff
@@ -6,18 +6,72 @@ const rl = readline.createInterface({
   input: process.stdin,
   output: process.stdout
 });
+const VOWELS = ['a', 'e', 'i', 'o', 'u']
+
+/*Function planning
+bool function isVowel
+  if vowel array.split
+  then array.push 'y' 'a' 'y' to the end
+bool function isConsonant
+  if consonant array.split
+  then array.pop[index0] consonant(s)
+  next array.push 'consonant(s)' 'a' 'y' to the end
+*/
+
+/*function isValid(str) {
+  const arr = [answer];
+  document.write(arr);
+  const letters = arr.split('');
+  vowels = ['a', 'e', 'i', 'o', 'u']
+  consonants = ['b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n', 'p', 'r', 's', 't', 'v', 'w', 'x', 'y', 'z']
+  for (index = 0; index < arr.length; index++) {
+    if (letters === !vowels || letters === !consonants) {
+      console.log('Please enter a valid word.');
+      return getPrompt();
+    }
+  }
+
+}*/
 
 
-function pigLatin(word) {
 
-  // Your code here
+// function isVowel(arr) {
+//   vowels = ['a', 'e', 'i', 'o', 'u']
+//   for (index = 0; index < arr.length; index++) {
+//     if (vowels.indexOf(arr[index]) === 0) {
+//       if (vowels.indexOf(arr[index]) === 1)) {
+//     return arr.push('yay');
+//     } else {
+      
+//     }
+//   }
+// }
 
+
+function pigLatin(word) {
+  word = word.toLowerCase().trim();
```

Comment:
we want to try and avoid mutating data, so store the formatted word into a new const and use that

Reply:
Ok! Made the correction. Thanks!